From 07abc70f2cf1d8ba28e4e307c6ed63d8218645f3 Mon Sep 17 00:00:00 2001
From: Jens Reidel <adrian@postmarketos.org>
Date: Thu, 2 Oct 2025 00:55:26 +0200
Subject: [PATCH] f0rmz: define shutdown method
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This would previous result in a warning:

../f0rmz/main.c:248:13: warning: ‘shutdown’ used but never defined
  248 | static void shutdown(void);
      |             ^~~~~~~~

and even fail to link on ppc64le.

Signed-off-by: Jens Reidel <adrian@postmarketos.org>
---
 f0rmz/main.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/f0rmz/main.c b/f0rmz/main.c
index 5e33332..6cbe9e3 100644
--- a/f0rmz/main.c
+++ b/f0rmz/main.c
@@ -275,6 +275,11 @@ static void toggle_theme_btn_clicked_cb(lv_event_t *event) {
     toggle_theme();
 }
 
+static void shutdown(void) {
+    sync();
+    reboot(RB_POWER_OFF);
+}
+
 static void sigaction_handler(int signum) {
     bbx_terminal_reset_current_terminal();
     exit(signum);
-- 
2.51.0

